test/setup/iam.tf

```diff
@@ -15,22 +15,90 @@
  */
 
 locals {
-  int_required_roles = [
-    "roles/bigquery.admin",
-    "roles/aiplatform.admin",
-    "roles/cloudfunctions.admin",
-    "roles/dataform.admin",
-    "roles/datalineage.viewer",
-    "roles/iam.serviceAccountAdmin",
-    "roles/iam.serviceAccountTokenCreator",
-    "roles/iam.serviceAccountUser",
-    "roles/logging.configWriter",
-    "roles/resourcemanager.projectIamAdmin",
-    "roles/run.invoker",
-    "roles/serviceusage.serviceUsageAdmin",
-    "roles/storage.admin",
-    "roles/workflows.admin"
-  ]
+  per_module_roles = {
+    root = [
```

Comment:
Root module is used by ADC. Can we have only roles which are needed by root module?